### PR TITLE
Variant

### PR DESCRIPTION
**Pull request by sudomake**
Added towards #22
```
Variant.Create
Variant.Put
Variant.Delete
```

### CODE REVIEW

File: variant_test.go [hunk 1]
```diff
@@ -5,9 +5,25 @@ import (
 	"testing"
 	"time"
 
+	"github.com/shopspring/decimal"
+
```
Comment (dlebech) [inline variant_test.go]: Minor nitpick: Could you remove this space? :)

**Comment by sudomake** [comment]
Sorry about that!  The space is auto-added by linters in both Atom and VS Code.

File: variant_test.go [hunk 2]
```diff
@@ -4,10 +4,25 @@ import (
 	"reflect"
 	"testing"
 	"time"
+	"github.com/shopspring/decimal"
```
Comment (dlebech) [inline variant_test.go]: Sorry for not being clear @sudomake . I meant the space between the two external dependencies, like so:

```
"time"

"github.com/shopspring/decimal"
httpmock "gopkg.in/jarcoal/httpmock.v1"
```
So we have the standard library imports grouped by themselves and external imports grouped by themselves.

File: variant.go [hunk 3]
```diff
@@ -3,7 +3,6 @@ package goshopify
 import (
 	"fmt"
 	"time"
-
```
Comment (dlebech) [inline variant.go]: This space shouldn't have been removed. Sorry if that was unclear. Please see my other comments. It's a bit pedantic of me to want this space :slightly_smiling_face:

**Comment by sudomake** [comment]
No worries, I appreciate your patience!

**Review by dlebech — APPROVED** [review]
Thanks @sudomake and sorry for the trouble :+1:

**Pull request merged**